packages/preview/miage-rapide-tp/0.1.0/template/main.typ

```diff
@@ -0,0 +1,84 @@
+#import "@local/miage-rapide-tp:0.1.0": *
+
+#show: doc => conf(
+  subtitle: "Rendu TP n°1",
+  authors: (
+    (
+      name: "NOM1 Prénom1",
+    ),
+    ( 
+      name: "NOM2 Prénom2",
+    ),
+    (
+      name: "NOM3 Prénom3",
+    ),
+    (
+      name: "NOM4 Prénom4",
+    ),
+  ),
+  "L3 MIAGE - Base de données",
+  date: "05/09/2024",
+  doc
+)
+
+= Introduction
+#lorem(10)
+
+= TP
+
+Une mini introduction pour le TP...
+#question("La question 1 ?")
+#remarque("Remarque personnalisée", bg-color: olive, text-color: white)
+
+#question("Donne un exemple de code en python.")
+Voici le code nécessaire pour afficher "Hello, World!" en Python :
+#code_block("print('Hello, World!')", "py", title: "HelloWorld en Python")
```

Comment:
The idiomatic casing convention in Typst is `kebab-case`, so I would recommend switching to that. But it is not a requirement.

Reply:
sorry about that, fixed it in 5e20354fa3f87bbbfc94b9644421295b39e9b980 !